src/components/Frame/Frame.tsx

```diff
@@ -368,14 +368,12 @@ export class Frame extends React.PureComponent<CombinedProps, State> {
 
   get getContext(): FrameContext {
     return {
-      frame: {
```

Comment:
What's your rationale behind removing this namespacing? I understand with the polaris namespace, but I'm not sure on this one

Reply:
It's to be inline with the new context api. With the old context api it would be `this.context.frame.startLoading` and with a consumer it would be
```tsx
<FrameContext.Consumer>
  {
    (frame) => {
      console.log(frame.frame.startLoading);
    }
  }
</FrameContext.Consumer>
```

And with the removal of `frame` it now becomes
```tsx
<FrameContext.Consumer>
  {
    (frame) => {
      console.log(frame.startLoading);
    }
  }
</FrameContext.Consumer>
```

`this.context` was already an object, containing all context, i.e this.context.polaris, this.context.frame, etc. We added frame so it wasn't `this.context.startLoading` however, we don't run into that anymore since we get the context from a consumer rather and the old context type system